### DISCUSSION TITLE
DIPY open group meetings

### DISCUSSION THREAD

**Discussion by skoudoro** [dipy/dipy#2398] [2. Announcements]
Hello @dipy/dipy-community! 

Below you can find the schedule for our open meetings. These include either exciting talks from a range of speakers or interesting software engineering discussions that aim to improve DIPY's upcoming versions.
 
The zoom link for the open meetings is https://iu.zoom.us/j/84926066336.

Be happy to subscribe to the [DIPY Calendar](https://calendar.google.com/calendar/u/0?cid=dXY4YzUwZmtmdnM1Mjk4MzdrMjk4dWVxaDBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) which is always updated with the new events or talks of the open meetings.

You are much welcome to request a talk or speaker that can be included in our upcoming open meetings schedule by sending an email to dipy@python.org.

Have in mind that we will be sharing some of these open meetings online at  [our YouTube channel](https://www.youtube.com/watch?v=1aHs21YAMWA&list=PLRZ9VSqV-6soOC0rUEAOV-QiSa_Qxk8JM)! 

| Date          | Time               | Speaker                    | Title                                        |
|---------------|--------------------|-----------------------------|----------------------------------------------|
|   December 16th, 2021 | 1pm EST / 7pm CET / 10am PT |  Eleftherios Garyfallidis: @Garyfallidis | Overview of 2021  |
|   December 2nd, 2021 | 1pm EST / 7pm CET / 10am PT |  Kurt Schilling: @schillkg | Tractography dissection variability due to the choice of segmentation methods |
|   November 18th, 2021 | 1pm EST / 7pm CET / 10am PT |  CANCELLED | CANCELLED |
|   November 4th, 2021 | 1pm EST / 7pm CET / 10am PT |  Ariel Rokem: @arokem | Dask Introduction and Integration in DIPY |
|   October 21rst, 2021 | 1pm EST / 7pm CET / 10am PT |  David Romero @drombas  | Atlasing of bundles |
|   October 7th, 2021 | 1pm EST / 7pm CET / 10am PT |  @dipy/dipy-community  | PR's Overview |
|   September  23rd, 2021 | 1pm EST / 7pm CET / 10am PT |  Jens Sjölund: [@jsjol] | Bayesian Uncertainity Quantification |
| September 9th, 2021 | 1pm EST / 7pm CET / 10am PT |    Eleftherios Garyfallidis: [@Garyfallidis ]   |   Fall Session Commencement   |
| May 13th, 2021       | 1pm EST / 7pm CET / 10am PT |               @skoudoro |                     Roadmap                      |
| April 29th, 2021     | 1pm EST / 7pm CET / 10am PT |             @skoudoro  @gabknight                  |                 Tractography contest, Tractography API  / Release overview                        |
| April 15th, 2021     | 1pm EST / 7pm CET / 10am PT |              ‪Leevi Kerkelä‬: @kerkelae                 |                      Q-Space Trajectory Imaging                         |
| April 1st, 2021     | 1pm EST / 7pm CET / 10am PT |              @Garyfallidis                 |                        Great workshop - Next steps - Community Feedback                       |
| March 18th, 2021      | 1pm EST / 7pm CET / 10am PT |    @dipy/dipy-community                 |   During  DIPY WORKSHOP March 15th-19 2021           | 
| March 4th, 2021      | 1pm EST / 7pm CET / 10am PT |    @skoudoro                      |         Release Preparation for upcoming workshop           |
| February 18th, 2021  | 1pm EST / 7pm CET / 10am PT |       @Garyfallidis            |    Workshop action items               |
| February 4th, 2021   | 1pm EST / 7pm CET / 10am PT |        Cancelled     |         Cancelled                          |
| January 21th, 2021   | 1pm EST / 7pm CET / 10am PT | Jelle Veraart: @jelleveraart | On axon diameter mapping using diffusion MRI |
| January 14th, 2021  | 1pm EST / 7pm CET / 10am PT | @Garyfallidis                 | Workshop preparation                         |

**Comment by skoudoro** [comment]
Hi @dipy/dipy-community,

This is just a reminder about the talk tommorrow by Jelle Veraart : On axon diameter mapping using diffusion MRI

See example paper https://elifesciences.org/articles/49855

link: https://iu.zoom.us/j/84926066336.

See you tomorrow all at 1pm EST / 7pm CET / 10am PT!

**Comment by arokem** [comment]
@Garyfallidis : could we please change this to use your IU zoom account? I have graduate student interviews on the 18th. I can still come to the DIPY meeting, but my UW zoom account will be in use at the same time.

**Comment by skoudoro** [comment]
here you are @arokem: https://iu.zoom.us/j/84926066336.

I updated all the link above

**Comment by skoudoro** [comment]
Hello @dipy/dipy-community,

Quick reminder for the meeting today at 1pm EST / 7pm CET / 10am PT! 

It should be a short meeting and the Subject will be DIPY Release. If you have a question, requirement about a PR, feel free to connect.

Cheers!

ps: same link as usual:  https://iu.zoom.us/j/84926066336

**Comment by arokem** [comment]
Sorry about this I've had an unavoidable conflict this morning, so I will have to miss this meeting. 😞

**Comment by skoudoro** [comment]
Hello @dipy/dipy-community,

Quick reminder for the meeting tomorrow at 1pm EST / 7pm CET / 10am PT! 

@Garyfallidis will talk about the future steps after the successful workshop. Feel free to come and ask your question, make some requests, etc. Be part of the future of DIPY!

See you tomorrow!

Cheers!

ps: same link as usual:  https://iu.zoom.us/j/84926066336

**Comment by skoudoro** [comment]
Hello @dipy/dipy-community,

Dr. Leevi Kerkelä (UCL) will discuss today (Thursday 15 April at 1pm EST / 7pm CET / 10am PT) his project on integrating Q-Space Trajectory Imaging in DIPY.

The zoom link for the open meeting is https://iu.zoom.us/j/84926066336.

We hope to see you online! 

Cheers !

**Comment by skoudoro** [comment]
Hello @dipy/dipy-community,

Quick reminder for the meeting today at 1pm EST / 7pm CET / 10am PT!

- @gabknight will talk about the new Tractography competition.
- We will have a brief talk about our Tracking framework
- @skoudoro will make a quick overview of the future Release. Let us know if there is any request.

Cheers!

ps: same link as usual: https://iu.zoom.us/j/84926066336

**Comment by skoudoro** [comment]
Hi @dipy/dipy-community,

Open group meetings in Spring 2021 reach their end.

We are preparing for the new season, Fall 2021, which will start in September 2021.

You can stay tuned by adding the DIPY event calendar to your agenda: https://calendar.google.com/calendar/u/0?cid=dXY4YzUwZmtmdnM1Mjk4MzdrMjk4dWVxaDBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ.

As usual, I am closing this issue and convert it into a discussion. 

Cheers!

**Comment by smaglioni** [comment]
Good morning, I missed the 9th September conference due to connectivity issues. If the conference was recorded, can you send me access to the recording?

**Comment by Garyfallidis** [comment]
Sure. Will get back to you asap. I have notes too.

**Comment by skoudoro** [comment]
[Here is the link to the paper](https://reader.elsevier.com/reader/sd/pii/S1053811918302696?token=9998898E4372592AA8F69028FBA1282DE464690AEEF32DDD4ADFCD2CC48BD772E1D6BEAD50921417FBF5AC9676012294&originRegion=us-east-1&originCreation=20210923172520)  for the `Bayesian Uncertainty Quantification` talk.
